docs/docs/about/references/keywords/TIME_SERIES.md

```diff
@@ -16,4 +16,34 @@ later referred to in the system of energy consumers defined under [INSTALLATIONS
 Reservoir variables and other time varying data not coming from a reservoir simulation model can
 be specified in a [CSV](https://en.wikipedia.org/wiki/Comma-separated_values) file.
 
+### Required attributes for TIME_SERIES
+
+| Attributes            | Description                                               |
+|-----------------------|-----------------------------------------------------------|
+| NAME                  | Time series reference name                                |
+| TYPE                  | Time series type. Either MISCELLANEOUS or DEFAULT.        |
+| FILE                  | Path to input file                                        |
+
+### Attributes dependent on TIME_SERIES TYPE
+
+| TYPE            | INTERPOLATION_TYPE                    | EXTRAPOLATION                          | INFLUENCE_TIME_VECTOR                                                                |
+|-----------------|---------------------------------------|----------------------------------------|--------------------------------------------------------------------------------------|
+| DEFAULT         | Not allowed to specify. RIGHT is used | Not possible to specify. FALSE is used | Optional. Default: TRUE                                                              |
+| MISCELLANEOUS   | Required: LEFT, RIGHT or LINEAR       | Optional. Default: FALSE               | Optional. Default: TRUE                                                              |
+| **Description** | Determine if time steps should contribute to global time vector. TRUE or FALSE. At least one time vector is required to be TRUE.| Defines whether the rates in the source should be set to 0 after the last time step (FALSE), or equal to value at last time step after the time interval (TRUE). | Defines how rates are interpolated between the given time steps (LEFT/RIGHT/LINEAR). |
```

Comment:
Switch first and last column here.

Reply:
I mean under description. The description under INTERPOLATION_TYPE explains INFLUENCE_TIME_VECTOR and the other way around.

Reply:
Nice catch! Done

Reply:
Btw: is it too much to include description in the table for TIME_SERIES. The explanation of these keywords will also be under the corresponding keywords, e.g. INTERPOLATION_TYPE. Maybe delete the description row? Now it looks like this:

<img width="894" alt="image" src="https://github.com/equinor/ecalc/assets/70878501/76693c07-0462-46ee-ba9e-8e45450725d8">

Reply:
I think it is a good idea to just remove the description here. And maybe also to flip the table, such that DEFAULT and MISCELLANEOUS are column headers, and each row corresponds to an attribute (INTERPOLATION_TYPE etc.)

Reply:
Talked to @halsyl, and she wanted the description row. Keep it for now?

Reply:
Sure. But I think we should flip the table so it is in line with the required attributes above. One attribute per line.

Reply:
Like this?
<img width="888" alt="image" src="https://github.com/equinor/ecalc/assets/70878501/5a51b7ee-cb03-4044-8cd9-6228a66bf4dc">